delfin/api/v1/storages.py

```diff
@@ -109,8 +111,8 @@ def create(self, req, body):
             capabilities = self.driver_api.get_capabilities(
                 context=ctxt, storage_id=storage['id'])
             validation.validate_capabilities(capabilities)
-            _create_performance_monitoring_task(ctxt, storage['id'],
-                                                capabilities)
+            self.perf_job_manager.create_perf_job(ctxt, storage['id'],
```

Comment:
Any reason why we move this  function to task manager.. API itself can create task table.. Why we need to distribute create task?

Reply:
`_create_performance_monitoring_task` only do one thing: add the job to db
`self.perf_job_manager.create_perf_job` do two things: add the job to db and put the task id to RabbitMQ